### ISSUE TITLE
Video display alternatives

### ISSUE THREAD

**Issue by brashrebel** [brashrebel/wptv#5]
I'm exploring ideas for alternative ways to present the videos in the contextual help menu. Trying to determine what is the ideal way to present the most appropriate and useful content to the user. Here are some concepts:

Current:
![screenshot 2015-04-28 21 07 05](https://cloud.githubusercontent.com/assets/4993131/7754505/174194f8-ffbb-11e4-8779-9685e2efa688.png)

Alternate 1:
![screenshot 2015-05-21 12 58 27](https://cloud.githubusercontent.com/assets/4993131/7754511/24a79a2a-ffbb-11e4-821d-189c88de4856.png)

Alternate 2:
![screenshot 2015-05-21 13 02 57](https://cloud.githubusercontent.com/assets/4993131/7754514/2c8d5fcc-ffbb-11e4-98b6-2b017acafdf4.png)

Alternate 3:
![screenshot 2015-05-21 13 08 16](https://cloud.githubusercontent.com/assets/4993131/7754516/33b5d680-ffbb-11e4-8834-86e318eb726b.png)

Alternate 4:
![screenshot 2015-05-21 13 31 05](https://cloud.githubusercontent.com/assets/4993131/7754856/ae16eaca-ffbd-11e4-8c67-fc4bc4ec6c81.png)

And of course there are plenty of other options. Thoughts?

**Comment by brashrebel** [comment]
My thoughts so far:

Current: Main issue is that there is no distinction between a "How to use this screen in WordPress" tutorial and a presentation or other style which is more like...bonus information if you want to learn more.
1. Still no labeling distinction but does separate them and emphasize one most important video.
2. Could be pretty labor intensive as each screen could have up to 3 topics and would all need to be defined in the plugin. Also, what do we do with screens that have more or less than 3 important topics?
3. Don't see any major issues here other than not showing other cool videos. However, there is a link and the other videos may simply not be super useful to those trying to get work done in the admin. Maybe focusing on a single video and giving a description would be best.
4. I like the idea of having a main video and links to video archives of related topics. Doesn't take full advantage of the real estate there though. But it does allow for us to present users with the option of learning more about many relevant topics for any particular screen which is kind of lacking in WP. Maybe instead of a bulleted list we could have...a tag cloud? Or...something else more visually appealing?

**Comment by jerrysarcastic** [comment]
> I like the idea of having a main video and links to video archives of related topics. Doesn't take full advantage of the real estate there though. 

I agree that it's a good idea too.  One thing to try is perhaps having two view states?  If we assume that there will always be one canonical video for each dashboard page (maintained by WPTV mods) then it makes sense to show an expanded view where there is only one video, like so:

![video-plugin_5a](https://cloud.githubusercontent.com/assets/1967448/8241087/0707a82c-15bc-11e5-8364-710c137ab61e.jpg)

We'd have some control over title length, format, description text, etc., so it's probably safe to design with these elements in mind.  For the `More Videos` link, since there is only one videos for this dashboard page, it can take a user to a more general location, like http://wordpress.tv/category/how-to/ or something like that.

If there are more related videos available (controlled by category on WPTV) then we can show a different view:

![video-plugin_5b](https://cloud.githubusercontent.com/assets/1967448/8241104/2b6b966a-15bc-11e5-8177-d25f8bcd6336.jpg)

Because the canonical video is in position 1, it still has hierarchy, and is likely the first video users will see.  I did not add any extra styling to it, but we could find a way to make it stand out if need be.  From there we fill in with other videos (how we select them is TBD) and the titles carry enough info that descriptions are not really needed.

In this view, the `More Videos` link can go to the category page for this dashboard view (if there is more than 2 related videos) or the main How-To videos page if there are no more related videos.

This adds a bit of complexity to the plugin, but I think from the user perspective it works nicely...

**Comment by brashrebel** [comment]
This is fantastic @jerrysarcastic (hey that rhymes)! Your mockups look amazing and I think nail perfectly what I both envision and believe is best. I'm not worried about the code complexity at all. Having two layouts depending on the screen and number of available videos is not _trivial_ but also not very challenging. I like it and will run with it.

Of course we can welcome input from everyone else but I really think this is a solid direction and we should focus our efforts elsewhere going forward. Thanks again!

**Comment by doctored** [comment]
i like it too! :)
